### PR TITLE
Add e2e tests for NamespaceManager

### PR DESCRIPTION
**Pull request by varvusc**

### CODE REVIEW

File: src/Microsoft.Azure.NotificationHubs/NamespaceManager.cs [hunk 1]
```diff
@@ -245,10 +246,12 @@ public async Task<NotificationHubDescription> CreateNotificationHubAsync(Notific
             };
 
             var token = _settings.TokenProvider.GetToken(uriBuilder.Uri.ToString());
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
```
Comment (iTOYS) [inline src/Microsoft.Azure.NotificationHubs/NamespaceManager.cs]: We already have one in ctor.

File: src/Microsoft.Azure.NotificationHubs/NamespaceManager.cs [hunk 2]
```diff
@@ -119,6 +119,7 @@ public NamespaceManager(IList<string> addresses, TokenProvider tokenProvider)
         ///                                      See <see cref="NamespaceManagerSettings.TokenProvider "/> to know more about the supported types.</exception>
         public NamespaceManager(Uri address, TokenProvider tokenProvider)
         {
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
```
Comment (mpodwysocki) [inline src/Microsoft.Azure.NotificationHubs/NamespaceManager.cs]: Is there a reason we are setting this explicitly?  Check the [TLS Guidelines for .NET](https://docs.microsoft.com/en-us/dotnet/framework/network-programming/tls) for more information on what we should be doing, which is to not hardcode TLS 1.2 if we can help it

Comment (varvusc) [reply src/Microsoft.Azure.NotificationHubs/NamespaceManager.cs]: removed.

Comment (varvusc) [reply src/Microsoft.Azure.NotificationHubs/NamespaceManager.cs]: it was locally issue

**Pull request merged**